src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.headerField {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-family: "Roboto", Arial, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.navList {
+  padding: 0;
+  display: flex;
+  list-style-type: none;
+}
+
+.navItems {
+  display: flex;
+  height: 60px;
+  margin-right: 20px;
+}
+
+.navItem {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: black;
+  height: 60px;
```

Comment:
don't hardcode with height, use paddings instead

Reply:
Added this height - as it is specified in the Figma layout.  Didn't I need to add this height?

Reply:
same - vertical paddings for links, but use integers (23px)

Reply:
Something like this?

.navItem {
  display: flex;
  align-items: center;
  text-decoration: none;
  color: black;
  padding: 23px 0; 
}